fog/ingest/enclave/impl/tests/tx_processing.rs

```diff
@@ -119,6 +121,7 @@ fn test_ingest_enclave(logger: Logger) {
             }
         }
 
+        log::info!(logger, "Line 125");
```

Comment:
That's line 124, not 125! :)
Should not be here regardless

Reply:
Changed it to 124 and then removed it :)